src/Homepage/index.js

```diff
@@ -147,6 +147,12 @@ class Homepage extends React.Component {
       : { pathname: "/location" };
   }
 
+  handleStoreFilterQuery = event => {
+    this.setState({
+      searchQuery: event.target.value
```

Comment:
we want the selection to be cleared once filtering begins, which is why `selectedLocation` is getting set to undefined. can we add that back?

Reply:
fixed